plugins/ledger/README.md

```diff
@@ -0,0 +1,76 @@
+![Bitcoin Cash](https://raw.githubusercontent.com/The-Bitcoin-Cash-Fund/Branding/master/Bitcoin_Cash/BCH%20Logo%20Long%20Text%20WhiteBG.png "")
+# Using Electron-Cash with Ledger Nano S
+
+Successfully tested with Ledger Nano S (1.3.1) Electron-Cash (3.1.2) and Bitcoin Cash Ledger App (1.1.5)
+
+---
+
+### Official Links
+[Product Website](https://www.ledgerwallet.com/products/ledger-nano-s)
+
+##### Chrome Applications
+[Ledger Manager](https://chrome.google.com/webstore/detail/ledger-manager/beimhnaefocolcplfimocfiaiefpkgbf)
+[Leger Wallet Bitcoin](https://chrome.google.com/webstore/detail/ledger-wallet-bitcoin/kkdpmhnladdopljabkgpacgpliggeeaf)
+
+##### Guides
+[User Guide](https://ledger.zendesk.com/hc/en-us/sections/115001453109-Ledger-Nano-S)
+[Ledger Bitcoin Cash App User Guide](https://ledger.zendesk.com/hc/en-us/sections/115001472725-Bitcoin-Cash)
+
+---
+
+### Requirements
+
+To complete a Bitcoin Cash transactions with a Ledger Nano S, you will need:
+
+1. An updated Ledger device - supported by Nano, HW.1, Nano S or Blue
+2. Bitcoin Cash app installed on your Ledger device
+3. "Browser Support" is disabled in the Bitcoin Cash Ledger app (only required for Nano S and/or Blue)
+
+PLEASE NOTE: Ensure that you do not have another application (like Electrum, Bitcoin Core, or any other software) opened on your computer
```

Comment:
Maybe clarify that other apps can run, as long as they are not connected to the ledger